src/tools/sdk/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Dart SDK
+description: Dart libraries and command-line tools.
+permalink: /tools/sdk
+js:
+- url: /tools/sdk/archive/assets/install.js
+  defer: true
+---
+
+The Dart SDK has the libraries and command-line tools that you need to develop
+Dart apps. To learn about other tools you can use for Dart development, see
+[Dart Tools](/tools).
+
+This site's documentation and examples use the
+[{{site.data.pkg-vers.SDK.vers}}][site SDK version]{:.no-automatic-external}
+release of the **Dart SDK**.
+
+## Install the SDK
+
+A package manager can help you easily install and update the Dart SDK.
+Don’t want to use a package manager? Other options are:
+
+* [Download the SDK manually](sdk/archive)
+* [Build the SDK from source][]
```

Comment:
I'm afraid this bulleted list might tempt people away from package managers. How about:

...update the Dart SDK. Other options are [building the SDK from source] and [downloading the SDK as a zip file].

That way the download link is still easy to find (last thing in the paragraph), but not quite as tempting.

Reply:
Good suggestion.